superset/migrations/versions/ae1ed299413b_add_timezone_to_report_schedule.py

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_timezone_to_report_schedule
+
+Revision ID: ae1ed299413b
+Revises: 3317e9248280
+Create Date: 2021-07-09 12:18:52.057815
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "ae1ed299413b"
+down_revision = "3317e9248280"
+
+import sqlalchemy as sa
+from alembic import op
+
+
+def upgrade():
+    op.add_column(
+        "report_schedule",
+        sa.Column(
+            "timezone", sa.String(length=100), nullable=False, server_default="UTC"
+        ),
+    )
```

Comment:
Older versions of SQLite do not support adding/dropping columns, so we use batch mode in cases like this (https://alembic.sqlalchemy.org/en/latest/batch.html).

```suggestion
    with op.batch_alter_table("report_schedule") as batch_op:
        batch_op.add_column(
            sa.Column(
                "timezone", sa.String(length=100), nullable=False, server_default="UTC"
            ),
        )
```